lib/workers/pr/index.ts

```diff
@@ -194,10 +201,7 @@ export async function ensurePr(
       const elapsedHours = Math.round(
         (currentTime.getTime() - lastCommitTime.getTime()) / millisecondsPerHour
       );
-      if (
-        !dependencyDashboardCheck &&
-        elapsedHours < config.prNotPendingHours
-      ) {
+      if (elapsedHours < config.prNotPendingHours) {
```

Comment:
I find this rearrangement of logic a little hard to follow. What was the reasoning?

Reply:
I wanted to make `Branch is ${elapsedHours} hours old - skipping PR creation` log message appear only when `prNotPendingHours` is in game.

Reply:
More broadly, it would be nice to have log messages for each reason, but maybe with separte PR